pkgs/data/fonts/arcticons-sans/default.nix

```
@@ -0,0 +1,29 @@
+{ fetchzip, lib, stdenvNoCC }:
```

```suggestion
{ lib
, stdenvNoCC 
, fetchzip
}:
```
small formatting thing